### PR TITLE
Add Professor Layton and the Miracle Mask

### PR DESCRIPTION
**Pull request by thedarknitro**
Adds Professor Layton and the Miracle Mask.

*Automatically generated by PR builder*

### CODE REVIEW

File: games/professor-layton-and-the-miracle-mask/game.dat [hunk 1]
```diff
@@ -0,0 +1,9 @@
+title = "Professor Layton and the Miracle Mask"
+description = "Professor Layton and the Miracle Mask is a puzzle adventure video game developed by Level-5. It is the fifth game in the Professor Layton series, the second in the prequel trilogy of games, and the first in the series to be developed for the Nintendo 3DS."
```
Comment (Leo626) [inline games/professor-layton-and-the-miracle-mask/game.dat]: Add the sentence about the Japanese only re-release.

> Professor Layton and the Miracle Mask Plus, an enhanced edition featuring new puzzles and added cutscenes, was released on the Nintendo eShop in Japan.

File: games/professor-layton-and-the-miracle-mask/game.dat [hunk 2]
```diff
@@ -1,9 +1,21 @@
...
+region = "eur"
+release_date = "2012-10-26"
+
+title = "0004000000031F00"
+region = "jpn"
+release_date = "2011-02-26"
+
+title = "00040000000C9200"
+region = "jpn"
+release_date = "2013-02-06" (Re-release)
```
Comment (Leo626) [inline games/professor-layton-and-the-miracle-mask/game.dat]: `(Re-release)` isn't suppose to be there. It was there so you would know it's a re-release.

**Comment by thedarknitro** [comment]
Okay, no problem. I took the screenshot using the Canary build. I print-screened the whole desktop, then cropped into MS Paint. Is there an easier way?

**Comment by Leo626** [comment]
```
- Image "../games/professor-layton-and-the-miracle-mask/screenshots/plmm-ingamenew.png"'s dimensions are 402 x 482 instead of the any of the following: 400 x 480
```
Travis is failing cause this screenshot is not 400 x 480. Remember to reduce CItra's window to the smallest size and use the Capture Region feature in ShareX to get the exact resolution.

Regarding the outlines, I believe you're using Canary which currently has a bug with Shader Emulation set to GPU. Change the Shader Emulation to CPU so that outlines render properly.

**Comment by thedarknitro** [comment]
I uploaded new screenshots. But the build is still failing because of the old screenshots, i think. How to delete the old ones?

**Pull request merged**